src/katello/enabled_report.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, repo_file=REPOSITORY_PATH):
         :type path: str
         """
         self.yb = yum.YumBase()
+        self.yb.preconf.debuglevel = 0
```

Comment:
This seems like a good change. What about making it dependent on the supress_debug setting? ie if supress_debug is True then set this to 0